src/main/java/in/projecteka/consentmanager/clients/ClientError.java

```diff
@@ -116,8 +116,8 @@ public static ClientError consentArtefactForbidden() {
                 new ErrorRepresentation(new Error(CONSENT_ARTEFACT_FORBIDDEN, "Cannot retrieve Consent artefact")));
     }
 
-    public static ClientError otpNotFound() {
-        return new ClientError(NOT_FOUND, new ErrorRepresentation(new Error(OTP_INVALID, "Invalid OTP")));
+    public static ClientError invalidOtp() {
+        return new ClientError(BAD_REQUEST, new ErrorRepresentation(new Error(OTP_INVALID, "Invalid OTP")));
```

Comment:
UnAuthorized is what swagger says If I'm not wrong.

Reply:
In that case what should be the error representation to app? 
Should it say invalid otp for both invalid and expired scenario.

Reply:
We have different error code each case no?